server/src/main/java/org/opensearch/index/store/Store.java

```diff
@@ -204,7 +206,12 @@ public Store(ShardId shardId, IndexSettings indexSettings, Directory directory,
         final TimeValue refreshInterval = indexSettings.getValue(INDEX_STORE_STATS_REFRESH_INTERVAL_SETTING);
         logger.debug("store stats are refreshed with refresh_interval [{}]", refreshInterval);
         ByteSizeCachingDirectory sizeCachingDir = new ByteSizeCachingDirectory(directory, refreshInterval);
-        this.directory = new StoreDirectory(sizeCachingDir, Loggers.getLogger("index.store.deletes", shardId));
+        this.directoryFileTransferTracker = new DirectoryFileTransferTracker();
```

Comment:
Can we initialize it in StoreDirectory itself? StoreDirectory can also have getter for this field and `Store.getDirectoryFileTransferTracker()` can delegate it to `StoreDirectory.getDirectoryFileTransferTracker`